elyra/pipeline/processor_airflow.py

```diff
@@ -73,26 +73,14 @@ def process(self, pipeline):
 
             self.log.debug("Uploading pipeline file: %s", pipeline_filepath)
 
-            github_client = Github(login_or_token=github_repo_token)
-            try:
-                # Upload to github
-                repo = github_client.get_repo(github_repo)
+            github_client = GithubClient(server_url=runtime_configuration.metadata.get('github_api_endpoint'),
```

Comment:
Could we collect the metadata instance value into a `github_api_endpoint` variable - when the other github-related values are collected, then reference that variable here?

Reply:
check and done.